cfgov/jinja2/owning-a-home/explore-rates/index.html

```diff
@@ -596,7 +596,7 @@ <h3 class="subhead">About our data source for this tool</h3>
         }, {
             "img": static("apps/owning-a-home/img/footer_loan-estimate.png"),
             "text": "Have a Loan Estimate?",
-            "label": "Review it with our tool",
+            "label": "Review loan estimate",
```

Comment:
It looks like these link labels are now only used for the image alt text. I'm not sure that's quite right. It seems like the link wrapping the image should get an `aria-label` with this text, and the image should have an empty `alt` attribute (indicating it's decorative). Rational being the `alt` should describe the image, not the link, and since this is the link description it should move to the parent link. The image is really just a decorative extension of the link adjacent to it.